homeassistant/components/google/calendar.py

```diff
@@ -138,15 +149,28 @@ async def async_setup_entry(
                     "has been imported to the UI, and should now be removed from google_calendars.yaml"
                 )
             entity_name = data[CONF_DEVICE_ID]
+            unique_id = f"{config_entry.unique_id}-{calendar_id}-{entity_name}"
+            # Migrate to new unique_id format which supports multiple config entries
```

Comment:
Maybe add a comment about when we added this migration? If we ever want to clean up the migration it's nice to know when it was added.